### PR TITLE
Added Legends: Arceus' move data

### PR DESCRIPTION
**Pull request by LOuroboros**
## Description
I'm not entirely sure about this PR since these moves lose a lot of essence and usability without LA's battle mechanics, but I'm submitting it anyway :eyes:
When/If said mechanics are implemented, a few changes will have to be made since moves in LA have varying accuracy, power, secondaryEffectChance, etc based on the style used when performing a move.

## **Discord contact info**
Lunos#4026

### CODE REVIEW

File: src/data/battle_moves.h [hunk 1]
```diff
@@ -13233,8 +13233,344 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_Z] =
...
+        .effect = EFFECT_PLACEHOLDER, // EFFECT_DIRE_CLAW,
+        .power = 60,
+        .type = TYPE_POISON,
+        .accuracy = 100,
+        .pp = 15,
+        .secondaryEffectChance = 0,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .flags = FLAG_MAKES_CONTACT | FLAG_PROTECT_AFFECTED | FLAG_MIRROR_MOVE_AFFECTED | FLAG_KINGS_ROCK_AFFECTED,
+        .split = SPLIT_PHYSICAL,
```
Comment (AsparagusEduardo) [inline src/data/battle_moves.h:13248]: Missing zMovePower.

File: src/data/text/move_descriptions.h [hunk 2]
```diff
@@ -3741,4 +3741,28 @@ const u8 *const gMoveDescriptionPointers[MOVES_COUNT - 1] =
...
+    [MOVE_BITTER_MALICE - 1] = gNotDoneYetDescription,
+    [MOVE_SHELTER - 1] = gNotDoneYetDescription,
+    [MOVE_TRIPLE_ARROWS - 1] = gNotDoneYetDescription,
+    [MOVE_INFERNAL_PARADE - 1] = gNotDoneYetDescription,
+    [MOVE_CEASELESS_EDGE - 1] = gNotDoneYetDescription,
+    [MOVE_BLEAKWIND_STORM - 1] = gNotDoneYetDescription,
+    [MOVE_WILDBOLT_STORM - 1] = gNotDoneYetDescription,
+    [MOVE_SANDSEAR_STORM - 1] = gNotDoneYetDescription,
+    [MOVE_LUNAR_BLESSING - 1] = gNotDoneYetDescription,
+    [MOVE_TAKE_HEART - 1] = gNotDoneYetDescription,
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3767]: Can you please add move descriptions?

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]

File: src/data/text/move_descriptions.h (hunk 2 above)
Comment (LOuroboros) [reply src/data/text/move_descriptions.h]: Done. A lot of the descriptions I wrote probably suck big fat oranges though, so suggestions are welcomed.

File: src/data/text/move_descriptions.h [hunk 3]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+    "Viciously spews flames at\n"
+    "a foe, becoming fixated.");
+
+static const u8 sWaveCrashDescription[] = _(
+    "Hurts the foe and the user,\n"
+    "but increases Action Speed.");
+
+static const u8 sChloroblastDescription[] = _(
+    "Hurts the foe and the user,\n"
+    "and lowers Action Speed.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3015]:
```suggestion
    "An blast that hurts the user\n"
    "and lowers its Action Speed.");
```

File: src/data/text/move_descriptions.h [hunk 4]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+
+static const u8 sMountainGaleDescription[] = _(
+    "The user hurls giant chunks\n"
+    "of ice to damage the foe.");
+
+static const u8 sVictoryDanceDescription[] = _(
+    "Dances to raise offenses,\n"
+    "defenses, and also damage.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3023]:
```suggestion
    "Dances to raise damage 50%,\n"
    "plus offenses and defenses.");
```

File: src/data/text/move_descriptions.h [hunk 5]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+
+static const u8 sHeadlongRushDescription[] = _(
+    "Charges the foe with a full-\n"
+    "body tackle. Lowers Defense.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3027]:
```suggestion
    "Hits with a full-body tackle,\n"
    "lowering the foe's Defense.");
```

File: src/data/text/move_descriptions.h [hunk 6]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+
+static const u8 sBarbBarrageDescription[] = _(
+    "Can poison on impact. Powers\n"
+    "up if the foe has an ailment.");
+
+static const u8 sEsperWingDescription[] = _(
+    "High crit. hit chance. Raises\n"
+    "the user's Action Speed.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3035]:
```suggestion
    "High critical hit ratio.\n"
    "Ups the user's Action Speed.");
```

File: src/data/text/move_descriptions.h [hunk 7]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+
+static const u8 sBitterMaliceDescription[] = _(
+    "Hurts a foe, harder if it has\n"
+    "an ailment. Can give FrstBte.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3039]:
```suggestion
    "Hurts a foe harder if it has\n"
    "an ailment. Can give FrstBte.");
```

File: src/data/text/move_descriptions.h [hunk 8]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+
+static const u8 sShelterDescription[] = _(
+    "The user hardens their skin\n"
+    "and obscures themself.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3043]:
```suggestion
    "The user hardens their skin\n"
    "and obscures itself.");
```

File: src/data/text/move_descriptions.h [hunk 9]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+
+static const u8 sTripleArrowsDescription[] = _(
+    "Shoots 3 arrows that up crit.\n"
+    "chance and lower foe's Defs.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3047]:
```suggestion
    "Ups critical-hit ratio for 3\n"
    "turns. Lowers foe Defenses.");
```

File: src/data/text/move_descriptions.h [hunk 10]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+
+static const u8 sInfernalParadeDescription[] = _(
+    "Hurts a foe, harder if it has\n"
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3050]:
```suggestion
    "Hurts a foe harder if it has\n"
```

File: src/data/text/move_descriptions.h [hunk 11]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+    "an ailment. May leave a burn.");
+
+static const u8 sCeaselessEdgeDescription[] = _(
+    "High crit. hit chance. Leaves\n"
+    "Splinters that hurt the foe.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3055]:
```suggestion
    "High critical hit ratio. Sets\n"
    "Splinters that hurt the foe.");
```

File: src/data/text/move_descriptions.h [hunk 12]
```diff
@@ -2978,6 +2978,102 @@ static const u8 sEerieSpellDescription[] = _(
...
+    "Hits with brutally hot sand.\n"
+    "May inflict a burn.");
+
+static const u8 sLunarBlessingDescription[] = _(
+    "The user heals themself and\n"
+    "conceals their presence.");
+
+static const u8 sTakeHeartDescription[] = _(
+    "The user raise spirit to heal\n"
+    "and strengthen themself.");
```
Comment (AsparagusEduardo) [inline src/data/text/move_descriptions.h:3075]:
```suggestion
    "The user lifts its spirits to\n"
    "heal and strengthen itself.");
```

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]

**Comment by AsparagusEduardo** [comment]
You can batch suggestions into a single commit, you know?

**Comment by LOuroboros** [comment]
> You can batch suggestions into a single commit, you know?

I had a feeling that was the case at some point, but I wasn't given the choice now for some reason.

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**